projects/canopy/src/lib/header/header.component.spec.ts

```diff
@@ -400,4 +404,80 @@ describe('HeaderComponent', () => {
       });
     });
   });
+
+  describe('with account menu', () => {
+    let tabKeyDownEvent: KeyboardEvent;
+    let focusSpy: jasmine.Spy;
+    let preventDefaultSpy: jasmine.Spy;
+
+    beforeEach(() => {
+      fixture = MockRender(`
+        <header lg-header>
+          <lg-header-logo src="http://a.b/logo.png" href="http://a.b"></lg-header-logo>
+
+          <lg-account-menu>
+            <lg-account-menu-list-item>
+              <button type="button" lgAccountMenuItem>
+                <lg-account-menu-item-label>Button</lg-account-menu-item-label>
+                <lg-icon name="radio-button-unselected"></lg-icon>
+                <lg-notification-badge count="3" accessText="You have 3 unread messages"></lg-notification-badge>
+              </button>
+            </lg-account-menu-list-item>
+            <lg-account-menu-list-item>
+              <a href="#" lgAccountMenuItem>
+                <lg-account-menu-item-label>Link</lg-account-menu-item-label>
+                <lg-icon name="radio-button-unselected"></lg-icon>
+              </a>
+            </lg-account-menu-list-item>
+          </lg-account-menu>
+        </header>
+      `);
+
+      component = fixture.componentInstance;
+
+      const logoEl = fixture.debugElement.queryAll(By.css('.lg-header-logo__link'))[0]
+        .nativeElement;
+
+      focusSpy = spyOn(logoEl, 'focus');
+      tabKeyDownEvent = new KeyboardEvent('keydown', { key: 'Tab', shiftKey: true });
+      preventDefaultSpy = spyOn(tabKeyDownEvent, 'preventDefault');
+
+      component.showResponsiveMenu = true;
+    });
+
+    it('prevents the default event', fakeAsync(() => {
```

Comment:
Do these methods need to be `fakeAsync`? I can't see why they would...but I'm a bit rusty on Angular these days so take that with a pinch of salt :D

Reply:
You are right, that was copied from another test, missed that. Will remove all the `fakeAsync` as it isn't needed in any of those tests